checkstyle-style.xml

```diff
@@ -212,7 +212,11 @@
     <property name="severity" value="error"/>
     <property name="lineSeparator" value="lf"/>
   </module>
-  <module name="SuppressionCommentFilter"/>
+  <module name="SuppressionCommentFilter">
```

Comment:
For consistency sake, I'm going to suggest that we just get rid of this rule. An IDE should quickly be able to fix the formatting on the other whitespace in comments.